src/components/Sidebar/__snapshots__/AppSidebar.spec.ts.snap

```diff
@@ -16,15 +16,18 @@ exports[`AppSidebar.vue renders snapshot 1`] = `
         <div
           class="is-menu-item nav-item"
           data-testid="home"
-          iconcustom=""
+          iconcustom="[object Object]"
           subnav="[object Object]"
         >
           <a>
             <div
               class="nav-icon"
             >
               
-              <!--v-if-->
+              <!-- eslint-disable-next-line vue/no-v-html -->
+              <div>
+                [object Object]
```

Comment:
```
[object Object]
```

seems to indicate that something was serialised that shouldn't have.

Reply:
This only happens in tests. You will find this in places where we use SVGs. During tests, SVGs are stubbed using `jest-transform-stub` and this happens to result in `[object Object]` being rendered. Since the SVGs are otherwise rendered fine, I don’t think this is an issue.